### PR TITLE
Adding Ruby - Rails useful Links and Resources

### PR DESCRIPTION
**Pull request by rishiip**

### CODE REVIEW

**Comment by rishiip** [comment]
@rathodchintan (cc: @PareshMayani ) Hi Chintan, Sorry to bother you, it would be great if you can tell the status of this PR!

**Comment by rathodchintan** [comment]
@rishiip, I checked your PR, but final merge will be done by @pareshmayani.

File: Rails/subscribe.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Name | URL
```
Comment (PareshMayani) [inline Rails/subscribe.md]: subscribe.md?? 

These 2 links should be added in "Useful_links" file, same as what we have done in Android newsletter, please check https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/Useful_links.md

And please delete this Subscribe file.

**Comment by PareshMayani** [comment]
@rishiip This PR looks good to me except a requested change! Please make a change and commit the requested change.

**Comment by rishiip** [comment]
@PareshMayani : Done! The PR is ready for merge :blush:

**Comment by PareshMayani** [comment]
Thank you @rishiip for the contribution! Really appreciate it.

**Comment by rishiip** [comment]
@PareshMayani Welcome :relaxed:

**Pull request merged**